doc/source/image_description/elements.rst

```diff
@@ -1233,6 +1233,58 @@ Used to customize the installation media images created for oem images
 deployment.
 For details see: :ref:`installmedia_customize`
 
+.. _sec.registry:
+
+<registry>
+----------
+
+Setup containers to fetch from one ore more registry elements
+
+.. code:: xml
+
+   <registry source="registry.opensuse.org">
+       <container name="some" use_with="podman"/>
+   </registry>
+
+The optional registry element specifies the location of one ore
```

Comment:
fixed

Reply:
Hmm the diff in GitHub still shows `ore` but this could be a GitHub diff artifact. I am aware that GitHub has occasional issues with displaying an accurate diff.